item-api/src/main/java/com/example/itemapi/global/config/LocalCacheConfig.java

```diff
@@ -0,0 +1,32 @@
+package com.example.itemapi.global.config;
+
+import com.github.benmanes.caffeine.cache.Caffeine;
+import org.springframework.cache.CacheManager;
+import org.springframework.cache.annotation.EnableCaching;
+import org.springframework.cache.caffeine.CaffeineCache;
+import org.springframework.cache.support.SimpleCacheManager;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+@Configuration
+@EnableCaching
+public class LocalCacheConfig {
+
+    @Bean
+    public CacheManager cacheManager() {
+        SimpleCacheManager cacheManager = new SimpleCacheManager();
+        List<CaffeineCache> caches = Arrays.stream(CacheType.values())
+                .map(it -> new CaffeineCache(it.getCacheName(), Caffeine.newBuilder().recordStats()
```

Comment:
```suggestion
                .map(it -> CaffeineCache.of(it))
```
현재는 cacheManager 함수 내에서 

카페인 객체와 캐시매니저 객체를 모두 생성하고 있는데요

각각 나눠서 카페인 캐시 객체를 생성하는 로직을 분리하면 더 가독성이 좋을것 같아요~!

Reply:
넵! 수정하도록 하겠습니다~